src/target/adiv5.c

```diff
@@ -731,13 +732,9 @@ void adiv5_dp_init(ADIv5_DP_t *dp)
 		}
 
 	}
-	if (dp->version == 0) {
+	if (dp->version == 0)
 		/* DP v0 */
-		DEBUG_WARN("DPv0 detected, no designer code available\n");
-		dp->designer_code = 0;
-		dp->partno = 0;
-		dp->mindp = 0;
-	}
+		DEBUG_WARN("DPv0 detected, designer detected from JTAG IDCode\n");
```

Comment:
we dont actually parse the designer from the idcode, so the comment isnt right, but we could